cards/CatalogEntry/1.json

```diff
@@ -1,19 +1,34 @@
 {
   "data": {
-    "type": "card",
     "attributes": {
       "title": "Person",
       "description": "Catalog entry for Person card",
       "ref": {
         "module": "http://local-realm/person",
         "name": "Person"
+      },
+      "demo": {
+        "firstName": "Mango",
+        "lastName": "Abdel-Rahman",
+        "isCool": false,
+        "isHuman": false,
+        "favorite": {}
```

Comment:
this seems like an odd value to include when the field should be empty. was this generated by our host, or did you hand code this? I would think that is we want to have an empty field we just set it to undefined instead of an empty object. If this was generated by the host app then probably this is a bug.

Reply:
Hmm I tried again now and it seems to be generated that way. I was thinking of handling fields like that as the next step after this PR, using the card picker to choose any non-primitive card there. But I can look into why it's being instantiated like that when no selection is made as well